src/fp.rs

```diff
@@ -12,7 +12,9 @@ pub(crate) const MAX_ROOTS: usize = 20;
 pub(crate) struct FieldParameters {
     /// The prime modulus `p`.
     pub p: u128,
-    /// `mu = -p^(-1) mod 2^64`.
+    /// `base = 2^64`.
```

Comment:
Is this `2^64` for all `FieldParameters`? If so, why have a struct field for it? If not:
```suggestion
    /// `base`, e.g., `2^64` for `FP128`.
```

Reply:
on a second though I removed the field.